crates/egui-winit/src/lib.rs

```diff
@@ -342,8 +342,8 @@ impl State {
             }
             WindowEvent::KeyboardInput { event, .. } => {
                 // When pressing the Tab key, egui focuses the first focusable element, hence Tab always consumes.
-                let consumed = self.on_keyboard_input(event)
-                    || self.egui_ctx.wants_keyboard_input()
```

Comment:
Here is the relevant diff of https://github.com/emilk/egui/pull/3649:

![image](https://github.com/emilk/egui/assets/1148717/a5c9e013-016c-4258-8067-6e2b551fbc05)

The change was was that some events were now marked consumed, namely cut/copy/past commands.

Is it these that are troubling you?

You change is a breaking change, not a revert.

If you want to revert to 0.24, you should ignore the result of `on_keyboard_input`, but your new code doesn't even call `on_keyboard_input`, which means the keyboard events won't make it to egui at all, which isn't great (the eframe app may be very interested in the key events, even though there is no TextEdit or similar listening)

Reply:
Sorry I'm a little confused. The new code **does** call `on_keyboard_input`, and I can see the clipboard events working fine when, for example, a textfield is focused. But when egui unconditionally consumes things when it doesn't need them, it's not possible to observe them outside of egui.

I agree that ignoring the result is closer to 0.24 but I figure this change was likely intentional, even if not documented, so I thought to keep it when egui flags that it wants them. Is that not correct?

Reply:
I agree with your diagnosis of the symptom, but your fix is wrong.

You code is `self.egui_ctx.wants_keyboard_input() && self.on_keyboard_input(event)` and will only call `on_keyboard_input` if `wants_keyboard_input()` is true, due to boolean short-circuting. This means egui won't be told of the events, which would be a new bug.

The correct fix would be to call `self.on_keyboard_input(event);` and ignore its return value, like we did in 0.24

Reply:
Alright, I'll revert it to how it was before that PR.

Reply:
Is this resolved then?